flatdata-rs/lib/src/generator.rs

```diff
@@ -60,14 +58,23 @@ use std::{
 /// // re-export if desired
 /// pub use my_schema::*;
 /// ```
-pub fn generate(
-    output_dir: &Path,
-    schemas_path: &Path,
-    generator: &Path,
-) -> Result<(), GeneratorError> {
-    let output_dir = output_dir.canonicalize()?;
+pub fn generate(schemas_path: &Path) -> Result<(), GeneratorError> {
```

Comment:
Do you think it makes sense to add another method, `generate_with_generator_path` which explicitly takes generator path and tries not to guess it? We also might want to document environment variables of this function.